client/src/main/java/com/arcbees/beestore/client/application/CurrentOrderImpl.java

```diff
@@ -30,11 +29,15 @@
 import com.google.web.bindery.event.shared.EventBus;
 
 public class CurrentOrderImpl implements CurrentOrder, HasHandlers {
+    private static final float TPS = 0.09975f;
+    private static final float TVQ = 0.05f;
```

Comment:
Yes, we already discussed that with @Elrhino . We said that for a showcase that would be fine, but you are right that it'd be more complicated than that for real. ;)